Fixed when game crashes, it doesn't log any time played

rileyrichard opened the pull request:
Fixed so that the game time is logged even if the game exits due to a crash or an abort.

rileyrichard committed:
fixed thing

rileyrichard committed:
forgot to sign

Signed-off-by: Riley Richard <bubbalou112@gmail.com>

Ryex commented:
oh, I though the DCO was fixed
@rileyrichard  See the [DCO Check report](https://github.com/PrismLauncher/PrismLauncher/pull/3392/checks?check_run_id=36749377252)
You'll need to either add a Remediation commit or rebase and edit every commit to be signed off.

rileyrichard commented:
> oh, I though the DCO was fixed @rileyrichard See the [DCO Check report](https://github.com/PrismLauncher/PrismLauncher/pull/3392/checks?check_run_id=36749377252) You'll need to either add a Remediation commit or rebase and edit every commit to be signed off.

i thought i added a new commit? im sorry ive never used DCO to sign off on anything

rileyrichard committed:
DCO Remediation Commit for Riley Richard <bubbalou112@gmail.com>

I, Riley Richard <bubbalou112@gmail.com>, hereby add my Signed-off-by to this commit: cb83f8e52db1046173a8def2f216cfcf1518b91f

Signed-off-by: Riley Richard <bubbalou112@gmail.com>

rileyrichard commented:
i believe i fixed the DCO errors

Trial97 commented:
Thanks
